pragma-node/src/infra/repositories/onchain_repository/publisher.rs

```diff
@@ -31,6 +31,7 @@ pub async fn get_publishers(
     let address_column = match network {
         Network::Mainnet => "mainnet_address",
         Network::Sepolia => "testnet_address",
+        Network::PragmaDevnet => "pragma_devnet_address",
```

Comment:
This is the code that creates the publisher table:
```sql
CREATE TABLE publishers (
    name VARCHAR NOT NULL,
    website_url VARCHAR NOT NULL,
    mainnet_address VARCHAR,
    testnet_address VARCHAR,
    publisher_type INTEGER NOT NULL CHECK (publisher_type IN (0, 1)) -- 0 = first party, 1 = 3rd party
);
```

There is no column `pragma_devnet_address` ; not sure we want to add it. If we would add such column, it would probably be a nullable column named `devnet_address`? @EvolveArt

Reply:
Yes let's add `pragma_devnet_address` as a NULLABLE VARCHAR